src/lib/ui/noise-sensor/NoiseSensorActivityList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+import type { NoiseSensorDevice } from 'seamapi'
+
+import { useEvents } from 'lib/seam/events/use-events.js'
+import { NoiseSensorEventItem } from 'lib/ui/noise-sensor/NoiseSensorEventItem.js'
+import { useNow } from 'lib/ui/use-now.js'
+
+interface NoiseSensorActivityListProps {
+  device: NoiseSensorDevice
+}
+
+export function NoiseSensorActivityList({
+  device,
+}: NoiseSensorActivityListProps): JSX.Element {
+  const date = useNow()
+  const [since] = useState(date)
+
+  const { events } = useEvents({
+    device_id: device.device_id,
+    event_type: 'noise_sensor.noise_threshold_triggered',
+    since: since.minus({ months: 1 }).toString(),
+  })
```

Comment:
```suggestion
  const now = useNow()
  const [mountedAt] = useState(now)

  const { events } = useEvents({
    device_id: device.device_id,
    event_type: 'noise_sensor.noise_threshold_triggered',
    since: mountedAt.minus({ months: 1 }).toString(),
  })
```